### ISSUE TITLE
Clarification on sequence features embeddings

### ISSUE THREAD

**Issue by maxime7770** [reczoo/FuxiCTR#129]
Hi,
First, thanks for the repo, great work!
I have a question about how embedding dimensions for sequence features are handled in the models. Taking DIN as an example (DIN.py file):

```
self.attention_layers = nn.ModuleList(
            [DIN_Attention(embedding_dim * len(target_field) if type(target_field) == tuple \
                           else embedding_dim,
                           attention_units=attention_hidden_units,
                           hidden_activations=attention_hidden_activations,
                           output_activation=attention_output_activation,
                           dropout_rate=attention_dropout,
                           use_softmax=din_use_softmax)
             for target_field in self.din_target_field])
```

It seems the model uses the default `embedding_dim` parameter (defined in the model configuration) for every sequence feature. However, shouldn’t it use the embedding dimension defined by the user in the dataset config? I believe the default `embedding_dim` should only be used if the user didn’t specify a dimension for a particular feature, correct?

For example, if I specify an embedding dimension of 64 for a sequence feature (in the dataset config) but use embedding_dim: 32 in the model config, I think there would be a mismatch because the attention layer would expect an input dimension of 32.

I used DIN as an example here, but I believe similar behavior might occur with other models in the repository.

Could you please confirm if this is the intended behavior or whether adjustments might be needed?

Thanks!

**Comment by zhujiem** [comment]
Yes, the current implementation of DIN only supports equivalent embedding dim. Modifications are required to set different embedding dims.
